.github/workflows/ci.yml

```diff
@@ -2,6 +2,9 @@ name: CI
 
 on:
   pull_request:
+    paths-ignore:
+      - 'docs/**'
+      - '.github/workflows/**'
```

Comment:
github workflows should not be added here, changes here may be potential problems

Reply:
Ah. I see. For example, change the test logic in workflows. I'll remove this line.